staging/publishing/rules.yaml

```diff
@@ -629,9 +629,22 @@ rules:
       branch: master
       dir: staging/src/k8s.io/kubectl
     name: master
+    dependencies:
+    - repository: api
```

Comment:
Can you also add kubectl to https://github.com/kubernetes/kubernetes/blob/master/staging/publishing/import-restrictions.yaml :)

Reply:
done, please check that I did that correctly, first time I've touched those files (lol this is my 2nd upstream PR) thanks :)

Reply:
> this is my 2nd upstream PR

Omg yay, welcome!! :tada: :tada:

Reply:
:blush